### PR TITLE
Added es version of bertology.mdx doc

### PR DESCRIPTION
**Pull request by jQuinRivero**
Fixes #15947

Added spanish version of language_modeling.mdx documentation file.

@omarespejel @sgugger

### CODE REVIEW

File: docs/source/es/bertology.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ayudar a la gente a acceder a las representaciones internas, principalmente adaptado de la gran obra de Paul Michel
+(https://arxiv.org/abs/1905.10650):
+
+
+- accediendo a todos los hidden-states de BERT/GPT/GPT-2,
+- accediendo a todos los pesos de atención para cada head de BERT/GPT/GPT-2,
+- adquiriendo los valores de salida y gradientes de las heads para poder computar la métrica de importancia de las heads y realizar la poda de heads como se explica
+  en https://arxiv.org/abs/1905.10650.
+
+Para ayudarte a entender y usar estas features, hemos añadido un script específico de ejemplo: [bertology.py](https://github.com/huggingface/transformers/tree/main/examples/research_projects/bertology/run_bertology.py) mientras extraes información y podas un modelo pre-entrenado en
```
Comment (omarespejel) [inline docs/source/es/bertology.mdx:35]:
```suggestion
Para ayudarte a entender y usar estas features, hemos añadido un script específico de ejemplo: [bertology.py](https://github.com/huggingface/transformers/tree/main/examples/research_projects/bertology/run_bertology.py) mientras extraes información y cortas un modelo pre-entrenado en
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

File: docs/source/es/bertology.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+(que algunos llaman "BERTología"). Algunos buenos ejemplos de este campo son:
+
+
+- BERT Rediscovers the Classical NLP Pipeline por Ian Tenney, Dipanjan Das, Ellie Pavlick:
+  https://arxiv.org/abs/1905.05950
+- Are Sixteen Heads Really Better than One? por Paul Michel, Omer Levy, Graham Neubig: https://arxiv.org/abs/1905.10650
+- What Does BERT Look At? An Analysis of BERT's Attention por Kevin Clark, Urvashi Khandelwal, Omer Levy, Christopher D.
+  Manning: https://arxiv.org/abs/1906.04341
+
+De manera de asistir al desarrollo de este nuevo campo, hemos incluido algunas features adicionales en los modelos BERT/GPT/GPT-2 para
```
[the remark this answers is not in the document]
Comment (jQuinRivero) [reply docs/source/es/bertology.mdx]: Already commited this. I don't know why it's not working properly.

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Muchas gracias @jQuinRivero for the PR! 🤗 Please let me know if you wish to translate another one. 

@sgugger LGTM :)

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**